content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -1118,3 +1118,8 @@ Each feature gate is designed for enabling/disabling a specific feature:
 
 * The [deprecation policy](/docs/reference/using-api/deprecation-policy/) for Kubernetes explains
   the project's approach to removing features and components.
+* Since Kubernetes 1.24, new beta APIs are not enabled by default.  When enabling a beta
+  feature, you will also need to enable any associated API resources.
+  For example, to enable a particular resource like
+  `storage.k8s.io/v1beta1/csistoragecapacities`, set `--runtime-config=storage.k8s.io/v1beta1/csistoragecapacities`.
```

Comment:
`/api/apistorage.k8s.io/v1beta1/csistoragecapacities/` is not a resource. I'd call that a collection, but you're overall enabling the API not just the **list** verb.

How about:
```suggestion
  feature, you will also need to enable at least one beta version of each associated API.
  For example, to enable the v1beta1 CSIStorageCapacity API, which has the URL path
  `/apis/storage.k8s.io/v1beta1/csistoragecapacities`, you add the command line argument
  `--runtime-config=storage.k8s.io/v1beta1/csistoragecapacities` to the kube-apiserver.
```

Reply:
Sometimes you have to an enable a beta API, but sometimes there is no new API type or it's simply a field on a GA API which is still enabled by default.  For instance, the server-side-apply feature has no API to enable.